File: src/codes/Users_Manual/geometry_3.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = new ControllerBuilder().AddDevice(new AUTD3(Vector3d.Zero)).Open(Nop.Builder());
var numDevices = autd.Geometry.NumDevices;
var numTransducers = autd.Geometry.NumTransducers;
var center = autd.Geometry.Center;

File: src/codes/Users_Manual/geometry_3.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder().add_device(autd3::AUTD3(autd3::Vector3::Zero())).open(autd3::link::Nop::builder());
const auto num_dev = autd.geometry().num_devices();
const auto num_tr = autd.geometry().num_transducers();
const auto center = autd.geometry().center();
//~return 0; }